src/rewriter/mod.rs

```diff
@@ -354,13 +354,15 @@ mod tests {
     use crate::test_utils::{Output, ASCII_COMPATIBLE_ENCODINGS, NON_ASCII_COMPATIBLE_ENCODINGS};
     use encoding_rs::Encoding;
     use itertools::Itertools;
-    use static_assertions::assert_impl_all;
     use std::convert::TryInto;
     use std::sync::atomic::{AtomicUsize, Ordering};
     use std::sync::{Arc, Mutex};
 
     // Assert that HtmlRewriter with `SendHandlerTypes` is `Send`.
-    assert_impl_all!(crate::send::HtmlRewriter<'_, Box<dyn FnMut(&[u8]) + Send>>: Send);
+    const _: () = {
+        const fn is_send<T: Send>() {}
+        is_send::<crate::send::HtmlRewriter<'_, Box<dyn FnMut(&[u8]) + Send>>>()
```

Comment:
Because we can avoid pulling in an entire crate to get exactly this code. Static assertions used to be very tricky in Rust and needed odd hacks, but since `const` eval has improved, the `static_assertions` crate is largely obsolete.

Reply:
This is a dev dependency so it's not adding any extra work for crates depending on lol-html. If I saw this code I would replace it with `static_assertions` and say something like "Simplified the const check by using `static_assertions`.".